### PR TITLE
Kill MO on port 8889 incase it is left running

### PR DESCRIPTION
**Pull request by ShaneHarvey**
We're seeing some weird failures where it looks like `mongo-orchestration stop` does not shutdown  MO:
```
 [2017/02/27 12:08:25.844] + mongo-orchestration stop
 [2017/02/27 12:08:26.264] ERROR: The process "2612" not found.
``` 

The next task to run on that machine may or may not work. Forcibly killing MO before starting it up should fix some failures.

Evergreen:https://evergreen.mongodb.com/version/58b4d5ed3ff1223783027a97

A patch that actually kills an existing MO instance:
https://evergreen.mongodb.com/task_log_raw/drivers_tools_tests_windows__os_windows~windows_64_vs2015_test_auth~noauth_ssl~nossl_test_3.2_sharded_cluster_4a1c22355e929daf80a641e6f1a4791a0255c231_17_02_27_23_27_11/0?type=T#L577

### CODE REVIEW

**Comment by ShaneHarvey** [comment]
@behackett

File: .evergreen/run-orchestration.sh [hunk 1]
```diff
@@ -63,11 +59,24 @@ elif virtualenv --system-site-packages venv || python -m virtualenv --system-sit
 fi
 cd -
 
-case "$OS" in
-  cygwin*)
-    ORCHESTRATION_ARGUMENTS="$ORCHESTRATION_ARGUMENTS -s wsgiref"
-    ;;
-esac
+ORCHESTRATION_ARGUMENTS="-e default -f $MONGO_ORCHESTRATION_HOME/orchestration.config --socket-timeout-ms=60000 --bind=127.0.0.1 --enable-majority-read-concern"
+if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
```
Comment (behackett) [inline .evergreen/run-orchestration.sh]: Where is OS set now?

Comment (ShaneHarvey) [inline .evergreen/run-orchestration.sh]: It's always set in cygwin. It seems to be the standard way to check for windows: https://github.com/mongodb-labs/drivers-evergreen-tools/pull/2#discussion_r90750514

Comment (behackett) [inline .evergreen/run-orchestration.sh]: Great. LGTM.

**Comment by behackett** [comment]
@ShaneHarvey I think this can me merged now?

**Comment by ShaneHarvey** [comment]
I don't push access yet: https://jira.mongodb.org/browse/OFFICEIT-13082

**Pull request merged**